projects/UWMadison_Parks.yaml

```
@@ -0,0 +1,12 @@
+Name: UWMadison_Parks
+Description: 
+Department: Nutritional Science
+FieldOfScience: Nutritional Science
+Organization: University of Wisconsin - Madison
```

```suggestion
Organization: University of Wisconsin-Madison
```
(shouldn't have the spaces around the dash)